platforms-wg/glossary/_index.md

```diff
@@ -1,43 +1,43 @@
 ---
-title: Glossary
-description: "Defines key terms used in the Platforms Working Group's writings."
+title: Glosario
+description: "Define términos clave usados en las escrituras del Platforms Working Group."
 ---
 
-See also: [Cloud Native Glossary](https://glossary.cncf.io/)
+Ver también: [Glosario Cloud Native](https://glossary.cncf.io/)
 
-If you wish to refer to these definitions outside the scope of the working group's papers, please note that they have been written in the context of both the CNCF and application delivery.
+Si deseas referirte a estas definiciones fuera del alcance de los documentos del grupo de trabajo, ten en cuenta que han sido escritas en el contexto de la CNCF y el despliegue de aplicaciones.
 
-## Platform
-A collection of capabilities, documentation, and tools that support developing, deploying, operating, and/or managing the delivery of products and services. A platform may include web portals, APIs, CLIs, protocol definitions, documentation, standards, and/or golden path templates. When done well, platforms enable faster and more reliable delivery of an organization's applications and services.
+## Plataforma (Platform)
+Colección de capacidades, documentación y herramientas que apoyan el desarrollo, despliegue, operativa y/o gestión de la entrega de productos y servicios. Una plataforma puede incluir portales web, APIs, CLIs, definiciones de protocolos, documentación, estándares y/o plantillas de "golden path". Cuando se hace bien, las plataformas permiten un despliegue más rápida y fiable de las aplicaciones y servicios de una organización.
 
-Depending on the scope and audience for a platform, it may sometimes be referred to as a “Developer Platform,” an “Internal Developer Platform (IDP),"  a “Delivery Platform,” an “App Platform,” or even a “Cloud Platform.” The term “Platform-as-a-Service (PaaS)” is also often used to describe platforms that are purchased or otherwise adopted from outside an organization, providing a more managed but often less customizable platform solution.
+Dependiendo del alcance y la audiencia de una plataforma, a veces se puede referir a ella como una "Plataforma de Desarrollo", una "Plataforma de Desarrollo Interno (IDP)", una "Plataforma de Entrega", una "Plataforma de Aplicaciones" o incluso una "Plataforma en la Nube". El término "Plataforma como Servicio (PaaS)" también se usa a menudo para describir plataformas que se compran o adoptan desde fuera de una organización, proporcionando una solución de plataforma más gestionada pero a menudo menos personalizable.
```

Comment:
"Dependiendo del alcance y la audiencia **para** una plataforma", it refers for the platform from the scope and audience perspective.